_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -27,6 +27,7 @@ In order to follow these instructions, you will need the following:
 
 - A second 3DS with custom firmware (the **source 3DS**) that is the same region as the 3DS you are trying to modify (the **target 3DS**)
   - The consoles must be USA, JPN, or EUR region consoles
+  - The source 3DS can be one [region changed](region-changing) to match the target 3DS
```

Comment:
`can be one` -> `can be`

If you look the other way, both are grammatically correct, but I feel that it should be worded so that "you can do this to make kartdlphax work" instead of "this one can be one that did that work so that kartdlphax works", if that makes sense?

Reply:
+1